docker-compose.yml

```diff
@@ -0,0 +1,75 @@
+version: "3"
+
+networks:
+  frontend:
+    external: true
+  app:
+    driver: bridge
+    internal: false
+    
+services:
+  mariadb:
+    image: itkdev/mariadb:latest
+    networks:
+      - app
+    ports:
+      - '3306'
+    environment:
+      - MYSQL_ROOT_PASSWORD=password
+      - MYSQL_USER=db
+      - MYSQL_PASSWORD=db
+      - MYSQL_DATABASE=db
+      #- ENCRYPT=1 # Uncomment to enable database encryption.
+
+  phpfpm:
+    image: itkdev/php7.2-fpm:latest
+    networks:
+      - app
+    environment:
+      - PHP_XDEBUG=${PHP_XDEBUG:-0}
+      - PHP_XDEBUG_REMOTE_AUTOSTART=${PHP_XDEBUG_REMOTE_AUTOSTART:-0}
+      - PHP_XDEBUG_REMOTE_CONNECT_BACK=${PHP_XDEBUG_REMOTE_CONNECT_BACK:-0}
+      - PHP_MAX_EXECUTION_TIME=30
+      - PHP_MEMORY_LIMIT=256M
+      - DOCKER_HOST_DOMAIN=${COMPOSE_DOMAIN}
+    depends_on:
+      - mariadb
+    volumes:
+      - .:/app:delegated
+    tmpfs:
+      - /app/var/cache:size=1G
+      - /tmp:size=1G
+
+  nginx:
+    image: nginx:latest
+    networks:
+      - app
+      - frontend
+    depends_on:
+      - phpfpm
+    ports:
+      - '80'
+    volumes:
+      - ${PWD}/.docker/vhost.conf:/etc/nginx/conf.d/default.conf:ro
+      - ./:/app:delegated
+    labels:
+      - "traefik.enable=true"
+      - "traefik.docker.network=frontend"
+      - "traefik.http.routers.${COMPOSE_PROJECT_NAME}.rule=Host(`${COMPOSE_DOMAIN}`)"
+
+  redis:
+    image: 'redis:4'
+    networks:
+      - app
+    ports:
+      - '6379'
+
+  elasticsearch:
+    image: 'elasticsearch:6.8.4'
```

Comment:
6.8.x is end of life on 2020-11-20. Go with 7.4.x?

Reply:
We stay as the 7.4.x simply failes the queries and app index create commands. And right now we need to get online fast.